tests/backup/backup_rbac_test.go

```diff
@@ -1166,6 +1435,8 @@ var _ = Describe("{VerifyRBACForAppAdmin}", func() {
 			scheduleBackupName, err := CreateScheduleBackupWithValidation(nonAdminCtx, userScheduleName, SourceClusterName, appAdminBackupLocationName, appAdminBackupLocationUID, scheduledAppContexts, make(map[string]string), orgID, preRuleNameMap[appAdminUser], preRuleUidMap[appAdminUser], postRuleNameMap[appAdminUser], postRuleUidMap[appAdminUser], periodicSchedulePolicyNameMap[appAdminUser], periodicSchedulePolicyUidMap[appAdminUser])
 			dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation and validation of schedule backup with schedule name [%s]", userScheduleName))
 			userBackupNamesMap[appAdminUser] = SafeAppend(&mutex, userBackupNamesMap[appAdminUser], scheduleBackupName).([]string)
+			err = suspendBackupSchedule(userScheduleName, periodicSchedulePolicyNameMap[appAdminUser], orgID, nonAdminCtx)
+			dash.VerifyFatal(err, nil, fmt.Sprintf("Suspending Backup Schedule [%s] for user [%s]", userScheduleName, appAdminUser))
```

Comment:
SuccessCheck?

Reply:
NamespaceLabelBackupSuccessCheck() isn't added here as it is a normal backup without namespace labelling and we make use of the method CreateScheduleBackupWithValidation() here for backup validation.
This step was already present before I made the changes. The reason I have added suspendBackupSchedule() here is because in case of NFS location, the deletion of backup was taking a lot of time and by then a new scheduled backup was getting created. This was causing the testcase to fail.